.changeset/moody-apples-hang.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+follow a pattern for errors that is putting them in errors-data
```

Comment:
```suggestion
Adds new content collections errors
```
Something like this. It looks like they are related to content collections?

Reply:
Fully got it, Thanks.